src/interactivity/win32/windowproc.cpp

```diff
@@ -855,7 +933,9 @@ void Window::_HandleWindowPosChanged(const LPARAM lParam)
 // - <none>
 void Window::_HandleDrop(const WPARAM wParam) const
 {
-    Clipboard::Instance().PasteDrop((HDROP)wParam);
+    const auto drop = reinterpret_cast<HDROP>(wParam);
+    Clipboard::Instance().PasteDrop(drop);
+    DragFinish(drop);

```

Comment:
Ah, that's a handle leak fix that accidentally sneaked in, probably because I initially used this branch for various experiments. I didn't even realize it was in here! I could revert it and submit it as a separate PR if you'd like.

Reply:
Oops, we shipped the handle leak in Windows. Oh well, you can leave this in here

Reply:
FWIW, and as far as I'm aware, the handle leak has existed for a long time now; it's not a recent regression.